### ISSUE TITLE
Discuss: Manage translations with Transifex

### ISSUE THREAD

**Issue by silopolis** [authoress/sphinx-intl-guide#4]
It would be nice to have the translations managed with Transifex as support is built-in sphinx-intl (hope other TMSs like Weblate will be added in the future...), furthermore if that is added to the guide (see #3 ).

@authoress how would you like it to be done at the organizational level ?
Do you already have an organization @Transifex ?
Would you mind creating one for your contents for us to contribute to ?
Would you need/want help to manage the projects in that organization ?

**Comment by authoress** [comment]
Yes, I'll definitely need help, never worked with Transifex and have no idea what GH organizations are for ))

**Comment by silopolis** [comment]
Le jeu. 24 nov. 2022 à 18:07, Natalia ***@***.***> a écrit :

> Yes, I'll definitely need help, never worked with Transifex
>
I only did as a translator ATM

have no idea what GH organizations are for ))
>
I meant Transifex orgs actually.
I'll have look at it and get back to you with actionnable infos...

**Comment by authoress** [comment]
@silopolis I think now I understand you)) Created an organisation and a project for sphinx-intl-guide on Transifex. What should I do now? Invite you somehow?

**Comment by silopolis** [comment]
Oh my, hat was 2 month ago 😱
Sorry 🙏
You may be able to invite me with the same username as here or witb my e-mail @gmail.com.
Or I can create a request if you give me the link to the Transifex project...

**Comment by authoress** [comment]
> Oh my, hat was 2 month ago 😱 Sorry 🙏 You may be able to invite me with the same username as here or witb my e-mail @gmail.com. Or I can create a request if you give me the link to the Transifex project...

Maybe the delay is for the better )) I'm going to add info about publishing on RTD/GH Pages (as you suggested in #5 ), and then you can translate. What do you think?

**Comment by silopolis** [comment]
Perfect 👌

**Comment by authoress** [comment]
@silopolis Hello Jérémie!
If you are still interested, the project is ready for translation, I sent you an invitation in Transifex. I've already added the pre-translated text, hope this saves you some time. Feel free to replace it with your own translation.

**Comment by silopolis** [comment]
@authoress Hey Natalia!
Hope you're doing well:)

I surely am!
Already started too many things ATM, so first have to make room in the pipe... But expect French translation before long, hopefuly soon 🤞 

TY
J

**Comment by authoress** [comment]
@silopolis Sure thing, take your time! Good luck with those many things ))
